ads/ix.js

```diff
@@ -49,52 +49,18 @@ export function ix(global, data) {
       doubleclick(global, data);
     };
 
-    data.targeting = data.targeting || {};
     if (typeof data.ixId === 'undefined' || isNaN(data.ixId)) {
       callDoubleclick(EVENT_BADTAG);
       return;
-    } else {
-      data.ixId = parseInt(data.ixId, 10);
-    }
-
-    if (typeof data.ixSlot === 'undefined' || isNaN(data.ixSlot)) {
-      data.ixSlot = 1;
-    } else {
-      data.ixSlot = parseInt(data.ixSlot, 10);
-    }
-
-    if (typeof global._IndexRequestData === 'undefined') {
-      global._IndexRequestData = {};
-      global._IndexRequestData.impIDToSlotID = {};
-      global._IndexRequestData.reqOptions = {};
-      global._IndexRequestData.targetIDToBid = {};
     }
 
+    global._IndexRequestData = undefined;
```

Comment:
We need it to exist in the global object passed to loadScript, though we may remove this in a future refactoring.